app/platform/fabric/Proxy.ts

```diff
@@ -188,6 +194,31 @@ export class Proxy {
 		return channel_genesis_hash;
 	}
 
+	getLatestBlockTime(channel, currentTime) {
+		let curBlockTime: any;
+		let agoTime = '';
+		if (!channel?.latestdate) return agoTime;
+		curBlockTime = channel.latestdate;
+		const dateCurBlock = new Date(curBlockTime);
```

Comment:
It will be good enough to give a meaningful name to the variables. like currentBlockDate